### DISCUSSION TITLE
Cannot read properties of undefined (reading 'computed')

### DISCUSSION THREAD

**Discussion by ghost** [gilbitron/laravel-vue-pagination#150] [Help & support]
### I´m getting this and i dont understand why, i did everything you put on the instructions:

#_Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'computed') laravel-vue-pagination.es.js?6bf0:226_

I also tried using you other component for vue 2 even though i´using vue 3, could you help me with this?

Regards.

**Comment by thiswallz** [comment]
I don't recommend you using this lib, has no movement for a lot of months, you will end up with a lot of problems if nobody maintains this.

**Reply by ghost** [reply]
Thanks for replying, so you suggest any other lib? I’ve been searching but this is the one most people use.

**Comment by jialeee17** [comment]
Hello, I also faced the same issue. May I know the reason?

**Comment by JcGuerrero15** [comment]
Same here. Any solution?
